example code is not working. SPI-00001: No implementation  type 'JDBC'

## Description

When I use the following configuration in my `config.yaml`:

```yaml
mode:
  type: Standalone
  repository:
    type: JDBC
```

I get the following error at runtime:

```
Caused by: org.apache.shardingsphere.infra.spi.exception.ServiceProviderNotFoundException: SPI-00001: No implementation class load from SPI 'org.apache.shardingsphere.mode.repository.standalone.StandalonePersistRepository' with type 'JDBC'.
```

## Steps to Reproduce

1. Generate code using the template:  
   [`Configuration.ftl` (Line 101)](https://github.com/apache/shardingsphere/blob/a4b557d4e484cca7e17e4cd4d905cc66df0b4c00/examples/shardingsphere-jdbc-example-generator/src/main/resources/template/java/config/Configuration.ftl#L101)
2. Use the generated config (`sharding--spring-boot-starter-jdbc--standalone--local`) as shown above.
3. Start the application.

## Expected Behavior

- The application should start successfully if the configuration is valid.
- Or, if `JDBC` is not supported, this should be clearly documented or the template should not generate such configuration.

## Actual Behavior

- The application fails to start with the above SPI exception.

## Additional Information

- [Link to the relevant code in Configuration.ftl](https://github.com/apache/shardingsphere/blob/master/examples/shardingsphere-jdbc-example-generator/src/main/resources/template/java/config/Configuration.ftl#L101)
- Version: (Please specify your ShardingSphere version)

## Suggestion

- Please clarify if `mode.repository.type: JDBC` is supported.
- If not supported, update the documentation and templates to avoid generating unsupported configurations.

--- a/examples/shardingsphere-jdbc-example-generator/src/main/resources/template/pom.ftl
+++ b/examples/shardingsphere-jdbc-example-generator/src/main/resources/template/pom.ftl
@@ -38,6 +38,13 @@
             <version>${r'${project.version}'}</version>
         </dependency>
     </#if>
+    <#if mode?contains("standalone") && repository == "JDBC">
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-standalone-mode-repository-jdbc</artifactId>
+            <version>${r'${project.version}'}</version>
+        </dependency>
+    </#if>
     <#if transaction?contains("xa")>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>

--- a/examples/shardingsphere-jdbc-example-generator/src/test/java/PomTemplateTest.java
+++ b/examples/shardingsphere-jdbc-example-generator/src/test/java/PomTemplateTest.java
@@ -1,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Test case for pom template generation.
+ */
+public class PomTemplateTest {
+
+    private Configuration templateConfig;
+
+    @BeforeEach
+    public void setUp() {
+        templateConfig = new Configuration(Configuration.VERSION_2_3_31);
+        templateConfig.setClassLoaderForTemplateLoading(getClass().getClassLoader(), "/template");
+    }
+
+    @Test
+    public void testStandaloneJdbcRepository() throws IOException, TemplateException {
+        Map<String, Object> dataModel = new HashMap<>();
+        dataModel.put("mode", "standalone");
+        dataModel.put("repository", "JDBC");
+        dataModel.put("feature", "sharding");
+        dataModel.put("framework", "spring-boot-starter-jdbc");
+        dataModel.put("transaction", "local");
+        dataModel.put("shardingsphereVersion", "5.5.3-SNAPSHOT");
+        
+        Template template = templateConfig.getTemplate("pom.ftl");
+        StringWriter writer = new StringWriter();
+        template.process(dataModel, writer);
+        
+        String result = writer.toString();
+        assertThat(result, containsString("shardingsphere-standalone-mode-repository-jdbc"));
+    }
+
+    @Test
+    public void testNoStandaloneJdbcRepository() throws IOException, TemplateException {
+        Map<String, Object> dataModel = new HashMap<>();
+        dataModel.put("mode", "standalone");
+        dataModel.put("repository", "File");
+        dataModel.put("feature", "sharding");
+        dataModel.put("framework", "spring-boot-starter-jdbc");
+        dataModel.put("transaction", "local");
+        dataModel.put("shardingsphereVersion", "5.5.3-SNAPSHOT");
+        
+        Template template = templateConfig.getTemplate("pom.ftl");
+        StringWriter writer = new StringWriter();
+        template.process(dataModel, writer);
+        
+        String result = writer.toString();
+        assertThat(result, !containsString("shardingsphere-standalone-mode-repository-jdbc"));
+    }
+}

